docs/synthetics/using-the-api.md

```diff
@@ -0,0 +1,777 @@
+title: Using the API.
+description: How to use the Synthetics API.
+
+In addition to scheduled monitor runs, you can also trigger monitor runs by using an API. You can use this API to trigger monitor runs as part of your [CI/CD pipeline](./ci-cd-integration.md) and block deployments if runs fail. When a run fails, you'll be alerted via your configured alert [notification hooks](../alerts/alert-notifications.md). The run monitor API can be used to:
```

Comment:
Probably want to add a bit of a more general sentence about the API, so that it is not focused solely on the run API.

Reply:
I wasn't sure if this was necessary or not. I will add it in.